VERSION

```diff
@@ -1 +1 @@
-0.10.3-dev
```

Comment:
why is this was called `dev` not `dev-1`?

Reply:
That was done https://github.com/infinyon/fluvio/commit/673e60c013f0f0e351ba335c5e7d462711b146b2

Reply:
that didn't follow the release doc.  @tjtelan  we need to automate this.